sdk/communication/Azure.Communication.PhoneNumbers/phone-numbers-livetest-matrix.json

```diff
@@ -56,7 +56,7 @@
         {
             "Agent": {
                 "Windows2022": {
-                    "OSVmImage": "MMS2022",
+                    "OSVmImage": "windows-2022",
```

Comment:
@benbp @mikeharder do you think that perhaps [this line](https://github.com/Azure/azure-sdk-for-net/blob/607a4bf8e7a3d56193bdab6a19118fc95f519452/sdk/communication/Azure.Communication.PhoneNumbers/phone-numbers-livetest-matrix.json#L71) also need an update?

>             "AZURE_TEST_AGENT": "WINDOWS_2019_NETCORE31"

Reply:
I have no idea how `AZURE_TEST_AGENT` is used, but yeah I'd update the OS and net versions to match reality.

Reply:
IIRC this was a workaround I and ACS came up with due to restrictions with registered phone numbers being used in live tests (1:1 limit), and being able to dynamically look up which phone number needed to be used from the keyvault secret.

FYI @danielortega-msft we may need to think about how to keep this variable updated, as if we rev the version here it won't be able to find the secret.

Reply:
@benbp @mikeharder for now I am going to resolve this conversation and merge this PR. If anybody has an idea if/how to update this string, I am happy to make another PR doing that.